src/pages/stats.astro

```diff
@@ -125,9 +147,23 @@ const factions = [
           ))
         }
       </div>
-      <div class="mt-4 min-w-0 flex-auto rounded-xl border border-gray-800 bg-gray-900 p-6">
-        <h2 class="flex-auto text-center text-2xl font-bold text-gray-600">More stats soon</h2>
-      </div>
+    </div>
+  </Section>
+  <Section title="Activity" class="!max-w-screen-md">
+    <div class="min-w-0 flex-auto rounded-xl border border-gray-800 bg-gray-900 p-4">
+      <h2 class="mb-4 text-xl font-black">Ladder games played</h2>
+      <GameNumberChart data={numberMatches} labels={dateLabels} client:idle />
+      <p class="mt-2 pl-1 text-xs leading-snug text-gray-500">
+        In total, {longNumberFormatter.format(sumMatches)} ladder games were played since {
+          earliestDate.toLocaleDateString("en", { dateStyle: "long" })
+        }.
+      </p>
+    </div>
+  </Section>
+  <Section class="!max-w-screen-md">
```

Comment:
I had to create another section otherwise they would be side by side (and too small in my case). 
I don't know what is best between this and allowing sections to always display in column?

Reply:
`<Section>` creates a flex layout. If your first element inside `<Section>` is a normal `<div>` without any classes you can stack the two charts on top of each other.

Also let's aim to use `<Widget>` or `<Box>` (from `layouts/Box.astro`) here?